src/rp2040/serial.c

```diff
@@ -3,21 +3,65 @@
 // Copyright (C) 2021  Kevin O'Connor <kevin@koconnor.net>
 //
 // This file may be distributed under the terms of the GNU GPLv3 license.
+//Modified 04/04/2024 by Amken3d (info@amken.us) to add all UART pin combos
```

Comment:
Please don't add changelog information to the code - use the git commit messages for describing and documenting history.  There's a brief note on this at https://www.klipper3d.org/CONTRIBUTING.html#what-to-expect-in-a-review .  Sorry - I missed this in the previous review.  Otherwise the changes look fine to me.

Thanks,
-Kevin